### DISCUSSION TITLE
Dead project?

### DISCUSSION THREAD

**Discussion by Abraka** [kanboard/kanboard#5188] [General]
Last release was nearly half year ago. Not much happening here. Is this project still alive or what?

**Comment by oliviermaridat** [comment]
Your answer is in the README file ;-)

Please also check commit list: https://github.com/kanboard/kanboard/commits/master

**Comment by Abraka** [comment]
Huh, indeed. Is there any live fork of kanboard or project sw to which people migrating from kanboard? Thx

**Comment by Kolossi** [comment]
I would suggest trying kanboard - i use it and as far as I am concerned it's feature-complete.

Others also continue to add functionality as plugins.

Why does it need to be a "live" fork if the software works? yes I do get in a production environment security patches are relevant, but if it's only exposed internally, maybe not such a big deal?

**Comment by Abraka** [comment]
Well I get your point, but thing is, if sw development is stopt or there are only some minor things changed in plugins...the sw project is effectively dead. Feature complete? There is gazzilion needed features to add, changes to be made in order to move this thing up to speed. Security patches...of course they needed...for every sw. 
This is SAD, Im using kanboard for many years and I always hoped that author will get his "s.it" together and bring this sw up to speed to todays world. Such a missed oportunity and potential. This could simply be a best opensource implementation of kanboard project sw around. Well anyway, thx to fgulliot for his time and work he put into this. 
Nothing last forever...

**Comment by Kolossi** [comment]
It has all the features I need from a simple kanban board piece of software, therefor as I said "as far as I am concerned it's feature-complete".  Perhaps @Abraka, you should take it over and get the "s.it" together.  It's open-source so (in line with the licence) you are free to fork and further develop.

Keeping a piece of software going with patches and updates is a massive undertaking, for which all of us have paid fgulliot precisely zero - and that's how much we are owed by them!  You didn't take the time to even read the readme they'd prepared before posting your Q!

Sorry to rant (I really am) but it ticks me off when people using free software that takes hours/days/weeks/years to develop then rag on the fact it isn't even better, and seem to think there's actually an obligation on the author to do not only more but do exactly what they personally need.

Fork the repo, do patches/fixes/improvements, then people may come over to your software ( and probably make multiple unreasonable demands on it and your time .... :-) )

**Comment by Abraka** [comment]
If it has all features you need...good for you.
No, I didnt read readme after so many years of using this sw. Sure, my bad.
Yea, must be really upseting, when somebody, who using your sw want more features and development...what?
I would love to fork repo, unfortunatelly my coding skills are near zero. But I love the classic argument in open source community...if you dont liked it or want more, you ungrateful bad, bad person-troll, just make your own sw.
So rant all you want, and let me do the same...this is free discussion after all :)

**Comment by Abraka** [comment]
1.2.21? Ok, Ok, I get it, not dead yet.
